source/guides/core-concepts/cypress-studio.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Cypress Studio
+---
+
+{% note info %}
+# {% fa fa-graduation-cap %} What you'll learn
+
+- How to extend tests interactively using the Cypress Studio
+{% endnote %}
+
+# Overview
+
+Cypress Studio provides a visual way to generate tests within the Test Runner, by *recording interactions* against the application under test.
+
+
+{% imgTag /img/guides/cypress-studio/cypress-studio-overview.png "Cypress Studio" "no-border" %}
+
+## Using Cypress Studio
+
+The Cypress {% fa fa-github %} {% url "Real World App (RWA)" https://github.com/cypress-io/cypress-realworld-app %} will be use to demo the functionality of Cypress Studio.
+
+### Extending a Test
+
+Create a file under `cypress/integration` with the following test scaffolding.
+
+```js
+describe('Cypress Studio Demo', function () {
+  beforeEach(function () {
+    // Seed database with test data
+    cy.task('db:seed')
+
+    // Login test user
+    cy.database('find', 'users').then((user) => {
+      cy.login(user.username, 's3cret', true)
+    })
+  })
+
+  it('create new transaction', function () {
+    // Extend test with Cypress Studio
+  })
+
+  it('create new bank account', function () {
+    // Extend test with Cypress Studio
+  })
+})
+```
+
+We will use Cypress Studio to perform a New Transaction user journey.
+
+We are logged into the application inside the `beforeEach` block.
+
+Cypress Studio is directly integrated with the {% url 'Command Log' test-runner#Command-Log %}.
+
+First, launch the Test Runner and run the spec created in the previous step.
+
+{% imgTag /img/guides/cypress-studio/extend-test-1.png "Cypress Studio" "no-border" %}
+
+Once the tests complete the run, hovering over the test in the Command Log will reveal an "Extend Test" button. Clicking on "Extend Test" will launch the Cypress Studio.
+
+{% imgTag /img/guides/cypress-studio/extend-test-2.png "Cypress Studio" "no-border" %}
+
+Click the "Get Started" button to begin interacting with your site to generate tests.
+{% imgTag /img/guides/cypress-studio/extend-test-3.png "Cypress Studio Get Started" "no-border" %}
+
+Next, the Test Runner will execute the test in isolation and pause after the last command in the test.
```

Comment:
In a green info block let's mention that Cypress will automatically execute all hooks and currently present test code, and then the test can be extended from that point on.